minutes/templates/minutes/detail.html

```diff
@@ -37,12 +37,12 @@
         <div class="btn-group btn-block mb-3">
             {% if prev %}
                 <a class="btn btn-outline-primary" href="{{ prev.get_absolute_url }}"><i
-                        class="fas fa-angle-left"></i><span class="sr-only">Previous</span></a>
+                        class="fas fa-angle-left"></i> <span>Previous</span></a>
```

Comment:
I think I would personally loose the icon altogether, but that's entirely up to personal taste, so feel free to ignore me

Reply:
Postponed decision until I sit down with a designer.